_layouts/page.html

```diff
@@ -5,7 +5,7 @@
 ---
 {% assign sidenav = site.data.navigation[page.sidenav] | default: page.sidenav %}
 {% if sidenav %}
-<aside class="usa-layout-docs-sidenav sidenav">
+<aside class="usa-layout-docs-sidenav sidenav {% if page.sticky_sidenav %}usa-sticky-sidenav{% endif %}">
```

Comment:
I think we should apply whatever logic we end up using here to the project and team-member page layouts as well, what do you think?

Reply:
Makes sense, waiting to hear back on whether this approach is acceptable before expanding :-)

Reply:
Added!

Reply:
Thanks!